packages/cardano-services/src/Asset/openApi.json

```diff
@@ -0,0 +1,119 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "UTxO",
```

Comment:
```suggestion
    "title": "Asset",
```

Reply:
Hi @rhyslbw ;

905559c21 solves this (and rebase as well)